core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -90,23 +90,23 @@ angular.module('oppia').component('contributionsAndReview', {
 
       var tabNameToOpportunityFetchFunction = {
         [SUGGESTION_TYPE_QUESTION]: {
-          [ctrl.TAB_TYPE_CONTRIBUTIONS]: () => {
+          [ctrl.TAB_TYPE_CONTRIBUTIONS]: shouldResetOffset => {
```

Comment:
Maybe add a JSDoc explaining what shouldResetOffset means and under what circumstances it should be true.